include/flecs/addons/json.h

```diff
@@ -84,13 +88,28 @@ const char* ecs_entity_from_json(
  *
  * @param world The world.
  * @param json The JSON expression to parse (see iterator in JSON format manual).
+ * @param desc Deserialization parameters.
+ * @return Last deserialized character, NULL if failed.
  */
 FLECS_API
 const char* ecs_world_from_json(
     ecs_world_t *world,
     const char *json,
     const ecs_from_json_desc_t *desc);
 
+/** Same as ecs_world_from_json, but loads JSON from file. 
```

Comment:
This should be `ecs_world_from_json()` with the parentheses so that doxygen links it correctly.

Reply:
Fixed!